src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -0,0 +1,413 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "HwndTerminal.hpp"
+#include <DefaultSettings.h>
+#include "../../renderer/base/Renderer.hpp"
+#include "../../renderer/dx/DxRenderer.hpp"
+#include "../../cascadia/TerminalCore/Terminal.hpp"
+#include "../../types/viewport.cpp"
+#include "../../types/inc/GlyphWidth.hpp"
+
+using namespace ::Microsoft::Terminal::Core;
+
+static LPCWSTR term_window_class = L"HwndTerminalClass";
+
+LRESULT CALLBACK HwndTerminalWndProc(
+    HWND hwnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam) noexcept
+{
+    return DefWindowProc(hwnd, uMsg, wParam, lParam);

```

Comment:
`DefWindowProcW`, right?

Reply:
I'd prefer W but its not a super big deal.